public/scripts/utils.js

```diff
@@ -17,3 +17,11 @@ export function $try(func) {
 export function reformatDatetime(dtStr) {
   return new Date(dtStr).toString();
 }
+
+export function downloadCSV(data, filename) {
```

Comment:
Are you suggesting that we'd just serve a CSV of the results of every prediction in the db?